app/views/devise/sessions/_mfa_enrolment.erb

```diff
@@ -1,29 +1,32 @@
-<h1 class="govuk-heading-xl"><%= t('mfa_enrolment.heading') %></h1>
-
 <div class="govuk-grid-row">
```

Comment:
looking at the designs, we should be using more than half columns, I suggest changing https://github.com/alphagov/verify-self-service/blob/83ae7c48a23826f4062a024b407aa998993d70d6/app/views/devise/sessions/new.html.erb#L4 to `two-thirds` or moving the grid responsibility to the partials...

Reply:
Agreed, currently I have it set up to use column-full. 

Here are the two styles

<img width="362" alt="Screen Shot 2019-10-17 at 17 45 56" src="https://user-images.githubusercontent.com/41922771/67029944-3d2d7d80-f106-11e9-9a7c-1ad1c5c16a38.png">

**Column-full**

<img width="310" alt="Screen Shot 2019-10-17 at 17 46 19" src="https://user-images.githubusercontent.com/41922771/67029945-3d2d7d80-f106-11e9-994f-b67e7fa74fff.png">

**Column-two-thirds**

Reply:
you need to leave full here, but change the half higher up in the layout. this is just a partial... in my previous comment i linked the parent div which needs to be made wider

Reply:
ahhhh, sorry...makes sense!